### ISSUE TITLE
[Serious] Visual list is not marked up as list

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#602]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87748 |
| **Checkpoint** | 1.3 Adaptable › Lists (1.3.1.f) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 02 Import/Export Workflow |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/12/26 at 12:03 pm |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

Content appears like a list but is not marked up as such.
In Export Settings, in each section, there are a list of checkboxes.  It would be nice to know how many checkboxes are in each section.

## Recommendation

Rule : 
Lists MUST be constructed using the appropriate semantic markup.

How To Fix: 
Fix this issue by using ONE of the following techniques:
1. Unordered list: Wrap a series of list items (<li>) inside an unordered list element (<ul>). Unordered lists should be used when a set of items can be placed in any order.

<ul>
<li>Strawberries</li>
<li>Papaya</li>
<li>Mangos</li>
</ul>

2. Ordered list: Wrap a series of list items (<li>) inside an ordered list element (<ol>). Ordered lists should be used when the list items need to be placed in a specific order.

<h3>How to boil an egg</h3>
<ol>
<li>Place eggs in a large saucepan.</li>
<li>Cover them with cool water by 1 inch.</li>
<li>Cover pan with a lid and bring water to a rolling boil over high heat.</li>
<li>When the water has reached a boil, remove saucepan from the burner.</li>
<li>Let eggs sit in water for 12 minutes.</li>
</ol>

3. When retrofitting legacy markup, using ARIA can be a useful way to add list semantics to markup without the risk of breaking any styles that might be attached to the legacy markup. Wrap the list items in a container that is marked with role="list". Mark each of the list items with role="listitem". NOTE: ARIA can be used to provide equivalent semantics for unordered lists only.

<div role="list">
<div role="listitem">Strawberries</div>
<div role="listitem">Papaya</div>
<div role="listitem">Mangos</div>
</div>

Reference: 
W3C-WAI tutorial: https://www.w3.org/WAI/tutorials/page-structure/content/#lists

Background: 
People who can see are able to look at a list and get a sense that it is a list, how large it is, and its structure - whether there are multiple lists, nested lists, etc. People who are blind do not have this ability if the list is not marked with semantic list markup. Lists must be marked semantically in a way that correctly identifies the list structure and type: unordered, ordered, or definition/description. When list markup is correctly applied, screen readers are able to notify users when they come to a list and tell them how many items are in a list.

## Screenshots

![02_lists](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87748/02_lists.png)
